src/terminal/input/terminalInput.hpp

```diff
@@ -87,7 +80,13 @@ namespace Microsoft::Console::VirtualTerminal
         bool _SearchWithModifier(const KeyEvent& keyEvent) const;
 
 
-    public:
+	public:
+        TerminalInput(_In_ std::function<void(std::deque<std::unique_ptr<IInputEvent>>&)> pfn);

```

Comment:
This class should have the other 4 default methods defined (move, copy, destructor, etc.). they can be made `= default;`

Reply:
The constructor cannot be defaulted, as it takes one argument. Visual Studios reports this as E1808.

Reply:
Should this also be done for `_TermKeyMap`?

Reply:
If the plan is to follow this [guideline](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c21-if-you-define-or-delete-any-default-operation-define-or-delete-them-all) then yes.

Reply:
Issue resolved.